src/Components/test/Ignitor/ElementNode.cs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Ignitor
+{
+    internal class ElementNode : ContainerNode
+    {
+        private readonly Dictionary<string, object> _attributes;
+        private readonly Dictionary<string, object> _properties;
```

Comment:
Is it really useful to distinguish between attributes and properties in this test infrastructure? What use will we make of that distinction?

Reply:
I'm not dead set against doing it as you have here, though. Just opening the question about what our goals are.

Reply:
Definitely, for this first pass I let the browser renderer drive the requirements of the intermediate representation (of which it set properties). If we find it's not useful we can nuke that later. This iteration is trying to work consistently with what the HTML browser renderer does.